requirements.txt

```
@@ -1,2 +1,3 @@
 numpy>=1.15
+pals-schema~=0.1
```

Oh, this would match anything `<1.0`.

If we want `>=0.1.1` and `<0.2` it must read:
```suggestion
pals-schema~=0.1.1
```